st2actions/st2actions/config.py

```diff
@@ -78,12 +78,13 @@ def _register_action_runner_opts():
     CONF.register_opts(ssh_runner_opts, group='ssh_runner')
 
     mistral_opts = [
-        cfg.StrOpt('v2_base_url', default='http://localhost:8989/v2',
-                   help='Mistral v2 API server root endpoint.'),
-        cfg.IntOpt('max_attempts', default=180,
-                   help='Maximum no of attempts made to connect to Mistral.'),
-        cfg.IntOpt('retry_wait', default=5,
-                   help='Time in seconds to wait before retrying connection to Mistral.')
+        cfg.StrOpt('v2_base_url', default='http://localhost:8989/v2', help='v2 API root endpoint.'),
```

Comment:
Yeah I saw that but there was a comment indicating known issue. I don't want to refactor this just yet as it's going to increase the scope of this PR. But I can work on it separately.